tests/compact/0007-context.jsonld

```diff
@@ -1,6 +1,6 @@
 {
   "@context": {
-    "dc": "http://purl.org/dc/elements/1.1/",
+    "dcterms": "http://purl.org/dc/elements/1.1/",
```

Comment:
That does not sound right: the url for dcterms shouldn't be `elements`

Reply:
(the same comment applies to all the files below...)

Reply:
I changed those in framing, but missed them here. Tests were using the prefix inconsistently.